CDTS Blazor/Pages/IdentityDocuments.razor

```diff
@@ -32,17 +32,44 @@
         </div>
     </form>
 
+    @if (completed == 1)
+    {
+        <div class="alert alert-success" role="alert">
+            Files uploaded successfully
+        </div>
+    }
+    else if (completed == 0)
+    {
+        <div class="alert alert-danger" role="alert">
+            Upload was unsucessful
+        </div>
+    }
+
     <a class="btn btn-primary" href="/signature" role="button">Next</a>
     <a class="btn btn-secondary" href="/" role="button">Cancel</a>
 </div>
 
 @code {
 
+    int completed = -1;
```

Comment:
Suggest using a nullable boolean and renaming to fileUploadComplete

Reply:
Now that I think about it maybe we want an enum with values UNKNOWN, UPLOAD_SUCCESS, UPLOAD_FAILED

Reply:
If I make it nullable I have to check if it has a value all the time